scripts/dockerfiles/Dockerfile.buildCNIPlugins

```diff
@@ -0,0 +1,28 @@
+# Copyright 2014-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+FROM golang:1.7
+MAINTAINER Amazon Web Services, Inc.
+
+# Check Volume Mounts
+RUN mkdir -p /go/src/github.com/aws/amazon-ecs-cni-plugins/bin/plugins
+VOLUME ['/go/src/github.com/aws/amazon-ecs-cni-plugins/bin/plugins']
+
+WORKDIR /go/src/github.com/aws/amazon-ecs-cni-plugins
+
+# TODO: Use go get
```

Comment:
I'd prefer to have source code retrieval happen outside `docker build`.  I also don't think this needs to be `go get`.